src/main/java/org/sagebionetworks/bridge/config/Environment.java

```diff
@@ -0,0 +1,12 @@
+package org.sagebionetworks.bridge.config;
+
+public enum Environment {
+    LOCAL,
+    DEV,
+    DEVELOP,
+    DEVELOPMENT,
+    STAGING,
+    UAT,
+    PROD,
+    PRODUCTION
```

Comment:
This seems like it could cause problems. Can we just go with the unabbreviated names of the four environments? It's worth changing code to consolidate.